app/android/src/uk/co/lutraconsulting/EXIFUtils.java

```diff
@@ -0,0 +1,126 @@
+package uk.co.lutraconsulting;
+
+import android.os.Bundle;
+import androidx.exifinterface.media.ExifInterface;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.FileInputStream;
+import java.io.OutputStream;
+import java.io.FileOutputStream;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import android.util.Log;
+
+public class EXIFUtils
+{
+    private static final String TAG = "EXIF Utils";
+    private static final int DEGREE_PRECISION = 2;
+    // GPS EXIF TAGS
+    private static final String GPS_BEARING_TAG = "GPSDestBearing";
+    private static final String GPS_BEARING_REF_TAG = "GPSDestBearingRef";
+    private static final String GPS_LON_TAG = "GPSLongitude";
+    private static final String GPS_LAT_TAG = "GPSLatitude";
+    private static final String GPS_DATE_TAG = "GPSDateStamp";
+    // Experimental TAGS
+    private static final String CAMERA_ELEVATION_TAG = "CameraElevationAngle";
+    private static final String GPS_ROLL_TAG = "GPSRoll";
+    private static final String GPS_PITCH_TAG = "GPSPitch";
```

Comment:
Can we have a brief description of these tags and their expected values, and what does it mean a tag is experimental?